### PR TITLE
Tidy stan

### PR DESCRIPTION
**Pull request by mortonjt**
This pull request
- Reparameterizes logits with log_inv_logit
- Adds zero avoiding log-normal priors
- Squeezes priors to reduce sampling errors
- Adds overwrite flag

### CODE REVIEW

**Comment by mortonjt** [comment]
This seems to be working, so merging
